src/core/dash_internal.h

```diff
@@ -13,7 +13,11 @@
 #include <cstring>
 #include <functional>
 
+#ifdef __s390x__
+#include <vecintrin.h>
+#else
 #include "core/sse_port.h"
+#endif
```

Comment:
May I suggest modifying `sse_port.h` to `#include <vecintrin.h>` instead of relying on the files including it?

Reply:
I'm sure after the change you suggested, something will break in the tests, so I suggest taking your suggestion to another PR.

Reply:
I feel like I need to explain my proposal, which really is a minor one:
Currently there are 2 files that `#include "core/sse_port.h"`. In your PR, you are first checking if the platform is s390x, and if so you do not include `sse_port.h` but include `<vecintrin.h>` instead. My proposal is to change `sse_port.h` to check which platform it is, and `#include <vecintrin.h>` if relevant.
This is a compilation change, so as long as it builds on all platforms it should not affect tests, right?
Is there anything that I'm missing?

Reply:
Done. 
The filename 'sse_port.h' is misleading because SSE is x86 technology. Can I rename the filename to 'simd_port.h'?